solution1/solution1.go

```diff
@@ -0,0 +1,360 @@
+package solution1
+
+import (
+	"bytes"
+	"context"
+	"crypto/tls"
+	"crypto/x509"
+	"encoding/json"
+	"fmt"
+	"io"
+	"io/ioutil"
+	"log"
+	"net/http"
+	"strings"
+	"time"
+
+	"github.com/giantswarm/operator-workshop/postgresqlops"
+)
```

Comment:
I split into multiple files, so this has only interesting code, and not e.g. os signal handling.